mozetl/symbolication/modules_with_missing_symbols.py

```diff
@@ -167,7 +167,10 @@ def are_symbols_available(debug_file, debug_id):
     for name, (version, debug_id, debug_file), count in top_missing
 ]
 
-subject = "Weekly report of modules with missing symbols in crash reports"
+subject = (
+    "Weekly report of modules with missing symbols in crash reports: %s"
+    % datetime.today().strftime("%Y-%m-%d")
```

Comment:
Should this use the `--date` parameter defined above?

Reply:
I added a note about how we should use `datetime.today()` as the script is currently built.